packages/ng/a11y/src/lib/skip-links/skip-links.component.html

```diff
@@ -0,0 +1,22 @@
+<nav role="navigation" id="top" class="skipLinks" [attr.aria-label]="'Goto'">
+	<ul class="u-listReset u-displayFlex skipLinks-list" role="presentation">
+		<li class="skipLinks-list-item">
+			<a
+				class="skipLinks-list-item-action button palette-grey mod-smaller"
+				href="#lucca-banner-solutions-container"
+				(click)="anchor('#lucca-banner-solutions-container', $event)"
+				>{{ 'Goto_Nav_Banner' }}</a
+			>
+		</li>
+		<li class="skipLinks-list-item">
+			<a class="skipLinks-list-item-action button palette-grey mod-smaller" href="#navSide" (click)="anchor('#navSide', $event)"
+				>{{ 'Goto_Nav_Navside' }}</a
+			>
+		</li>
+		<li class="skipLinks-list-item">
+			<a class="skipLinks-list-item-action button palette-grey mod-smaller" href="#main-content" (click)="anchor('#main-content', $event)"
+				>{{ 'Goto_Content' }}</a
+			>
+		</li>
+	</ul>
+</nav>
```

Comment:
If we need to add extra skip links on a specific app or a section within an app, we won't be able to use this component. We could pass an array of `skipLink` objects for the component to loop over to make the component more customisable.

Reply:
Yes, it can be an evolution!
But I think we can afford to move forward with this simple version before going further.